### PR TITLE
KEYCLOAK-3365 Configure required actions for LDAP in READ_ONLY mode

### PR DESCRIPTION
**Pull request by sventorben**

### CODE REVIEW

**Comment by mposolda** [comment]
@sventorben Sorry for late reply. The PR looks good, but there is test missing. Do you please have a chance to add test method for this? Probably to the LDAPReadOnlyTest class?

**Comment by sventorben** [comment]
@mposolda I have added a simple test case. Is that sufficient?

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPReadOnlyTest.java
Comment (mposolda) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPReadOnlyTest.java:133]: Do you please have a chance to update this test a bit and test that requiredAction add/remove works? As I believe that is the main reason why you created this PR. I mean something similar to this:
```
        UserResource user = ApiUtil.findUserByUsernameId(testRealm(), "johnkeycloak");
        UserRepresentation userRepresentation = user.toRepresentation();
        user.setRequiredActions(Collections.singletonList(UserModel.RequiredAction.CONFIGURE_TOTP.toString());
        user.update(userRepresentation);

        // Check and revert required action
        userRepresentation = user.toRepresentation();
        Assert.assertEquals(user.toRepresentation().getRequiredActions().getSize(), 1);
        Assert.assertEquals(user.toRepresentation().getRequiredActions().get(0), UserModel.RequiredAction.CONFIGURE_TOTP.toString());
        user.setRequiredActions(Collections.emptyList());
        user.update(userRepresentation);
```

**Review by mposolda — CHANGES_REQUESTED** [review]
@sventorben Thanks! I propose to update a test a little bit and I think  we will be good to go then.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPReadOnlyTest.java
Comment (sventorben) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPReadOnlyTest.java:135]: Good point! Done.

**Review by mposolda — APPROVED** [review]
@sventorben Thanks! I am approving, but can't merge right now because we have some temporary issues with travis/CI. Will merge once it is fixed.. Thanks for the patience.

**Pull request merged**
